crates/wash/src/cli/cmd/dev/mod.rs

```diff
@@ -255,6 +255,7 @@ pub async fn handle_command(
     };
 
     // See if the host is running by retrieving an inventory
+    std::thread::sleep(std::time::Duration::from_secs(1));
```

Comment:
Currently this runs into a timing issue on the function call below. I am guessing because inferring the ID is much faster than parsing logs, and that the host is not yet up by the time this function call is reached. It works with a sleep (a much smaller sleep is enough).

Is there a simple way to check whether a host is up? Or should I just implement a retry loop on the inventory retrieval?